### PR TITLE
Release 2.1

### LINKED ISSUE #340: Commonalities r2.1
**Problem description**
For milestone **M2** Commonalities should publish Release Candidate 

**Expected action**
The release PR for r2.1 needs to be prepared, when scope of the version 0.5 (documented in #273) is covered.

**Additional context**
For meta-release _Spring25_ the target date for M2 is planned for **7th of December 2024**: https://lf-camaraproject.atlassian.net/wiki/spaces/CAM/pages/14560849/Meta-release+Spring25#Milestone-status

### PR DESCRIPTION
**Pull request by rartych**
#### What type of PR is this?

* subproject management

#### What this PR does / why we need it:

This is Release PR for release 2.1 containing version v0.5.0-alpha.1 of Commonalities

#### Which issue(s) this PR fixes:

Fixes #340 

#### Does this PR introduce a breaking change?

- [x] Yes, some of the included PR in the release are having breaking changes
- [ ] No

#### Changelog input

```
r2.1 containing version v0.5.0-alpha.1 of Commonalities

```

### CODE REVIEW

**Comment by rartych** [comment]
The description of PR# https://github.com/camaraproject/Commonalities/pull/329 will be added before final review

**Comment by rartych** [comment]
Ready for final review

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,52 @@
...
+
+### Changed
+* Normalization of error status and code allowed values using `enum` by @PedroDiez in https://github.com/camaraproject/Commonalities/pull/316
+* Guidelines for subscription and event notification in API Design Guideliness 0.5.0 by @bigludo7 in https://github.com/camaraproject/Commonalities/pull/313 main changes:
+  - updated `terminationReason` in event notification type "subscription-ends"
+  - updated description of `sink` and `sinkCredential` attributes for subscription
+  - added rules for subscriptions with device identifier attribute
+  - added section 11.7 Resource access restriction relevant to subscriptions
+  - added clarification on `expiresAt` attribute for subscription
+* Updated error codes and changed `info.description` template for device / phone number identifiers in Appendix A in API Design Guideliness by @eric-murray in https://github.com/camaraproject/Commonalities/pull/324
```
Comment (PedroDiez) [inline CHANGELOG.md:39]: Suggestion:

* Updated error codes and changed `info.description` template for device / phone number identifiers in Appendix A in API Design Guideliness by @eric-murray in https://github.com/camaraproject/Commonalities/pull/324 and https://github.com/camaraproject/Commonalities/pull/346

Comment (rartych) [reply CHANGELOG.md]: resolved in https://github.com/camaraproject/Commonalities/pull/342/commits/acfbc56e8025ea10b4294f310a79637c2ecd0503

File: CHANGELOG.md [hunk 2]
```diff
@@ -24,20 +24,20 @@ This release contains documents and artifacts of Commonalities version 0.5.0-alp
...
 * Security scheme added to CAMARA_common.yaml by @rartych in https://github.com/camaraproject/Commonalities/pull/335
-* VERSION.yaml file added to indicate Commonlities versionby @rartych in https://github.com/camaraproject/Commonalities/pull/339
+* VERSION.yaml file added to indicate Commonalities version by @rartych in https://github.com/camaraproject/Commonalities/pull/339
 * Filtering for boolean guideline and examples in API Design Guidelines by @rartych in https://github.com/camaraproject/Commonalities/pull/336
 * Guidelines on the coverage of error codes in API-Testing-Guidelines by @jlurien in https://github.com/camaraproject/Commonalities/pull/343
 
 ### Changed
 * Normalization of error status and code allowed values using `enum` by @PedroDiez in https://github.com/camaraproject/Commonalities/pull/316
-* Guidelines for subscription and event notification in API Design Guideliness 0.5.0 by @bigludo7 in https://github.com/camaraproject/Commonalities/pull/313 main changes:
+* Guidelines for subscription and event notification in API Design Guideliness by @bigludo7 in https://github.com/camaraproject/Commonalities/pull/313 main changes:
```
Comment (PedroDiez) [inline CHANGELOG.md:33]:
```suggestion
* Guidelines for subscription and event notification in API Design Guidelines by @bigludo7 in https://github.com/camaraproject/Commonalities/pull/313 main changes:
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,14 +1,52 @@
...
+  - updated `terminationReason` in event notification type "subscription-ends"
+  - updated description of `sink` and `sinkCredential` attributes for subscription
+  - added rules for subscriptions with device identifier attribute
+  - added section 11.7 Resource access restriction relevant to subscriptions
+  - added clarification on `expiresAt` attribute for subscription
+* Updated error codes and changed `info.description` template for device / phone number identifiers in Appendix A in API Design Guideliness by @eric-murray in https://github.com/camaraproject/Commonalities/pull/324 and https://github.com/camaraproject/Commonalities/pull/346
+* Guidelines regarding mandatory error `status` and alignment of error codes related to identifiers in API Design Guideliness by @PedroDiez in https://github.com/camaraproject/Commonalities/pull/329 and https://github.com/camaraproject/Commonalities/pull/351
```
Comment (PedroDiez) [inline CHANGELOG.md:40]:
```suggestion
* Guidelines regarding mandatory error `status` and alignment of error codes related to identifiers in API Design Guidelines by @PedroDiez in https://github.com/camaraproject/Commonalities/pull/329 and https://github.com/camaraproject/Commonalities/pull/351
```

**Review by PedroDiez — COMMENTED** [review]
Minor points

**Review by PedroDiez — APPROVED** [review]
LGTM

**Review by hdamker — APPROVED** [review]
LGTM - thanks for all the work.

For some of the listed PRs I'm not sure if the statement "no breaking change" is correct (from perspective of impacted API specifications). Have added comments there.

**Review by patrice-conil — APPROVED** [review]
LGTM

**Pull request merged**